scapy/tools/UTscapy.py

```diff
@@ -517,10 +517,16 @@ def import_UTscapy_tools(ses):
     ses["Bunch"] = Bunch
 
 
+scapy_session = None
```

Comment:
Could you move this definition at the top the file?

Reply:
Done